src/screens/Exchange/ExchangeConfirm.js

```diff
@@ -177,21 +169,8 @@ class ExchangeConfirmScreen extends React.Component<Props, State> {
   }
 
   fetchTransactionEstimate = async () => {
-    const { activeAccountAddress, isSmartAccount } = this.props;
-    const { transactionSpeed } = this.state;
-    const txSpeed = transactionSpeed || SPEED_TYPES.NORMAL;
     this.setState({ gettingFee: true });
-
-    let gasLimit;
-    if (!isSmartAccount) {
-      gasLimit = await calculateGasEstimate({ ...this.transactionPayload, from: activeAccountAddress });
-      this.setState({ gasLimit });
-    }
-
-    const txFeeInfo = isSmartAccount
-      ? await this.getSmartWalletTxFee()
-      : this.getKeyWalletTxFee(txSpeed, gasLimit);
-
+    const txFeeInfo = await this.getSmartWalletTxFee();
```

Comment:
Could we add a .catch clause here and report to Sentry please? We tend to have issues around this functionality (not necessarily the app's fault but it's just for visibility)

Reply:
@jdnmr this should be replaces with new estimate flow.